news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+While they are still trying to find their own voice in the scene, it will be quite fascinating to see how guden will develop as a mapper. He is always willing to try something new and has already been making great strides in improving.
+
+### MJHs (osu!mania)
+
+[MJHs](https://osu.ppy.sh/users/2288943) is a Korean mapper who’s been in and out of the mania mapping community since late 2018, but has been getting a bit of traction as of recently within the anti-meta keymode community, primarily 5K. A lot of his maps from the start of his mapping career were pretty straightforward and simplistic to say the least, however he’s been notably trying new and innovative concepts to implement into his more newer maps that definitely make him stand out amongst a crowd of 5K mappers. And though the 5K community is very small to begin with, his dedication to the keymode is unwavering to say the least and it’s why this small, niche spot in the community is why many of his maps are starting to hold amazingly in the current 5K mapping scene, despite not having any ranked maps as of now.
+
+One of his most solidified maps in his mapping discography is [Konami Amusement (Wakita Jun) - Vide Nostalgia](https://osu.ppy.sh/beatmapsets/1197900#mania/2530522), which not only displays some excellent timing craftsmanship, but as well as heavy PR patterning whilst maintaining mapping standards that fit along with the song beautifully. The map also features a constant 3/4 timing signature that can somewhat throw players off from your normal 4/4 timing signature that many are accustomed to, making this quite a challenge to acc. The mapset as a whole resembles MJH’s current mapping style perfectly, which is why it holds such an important place in his mapping discography. 
+
+Another mapset of his that is rather new than some of older mapsets, but still an amazing mapset anyhow is [Akaru muu - Choukousoku henbyoushi no alien](https://osu.ppy.sh/beatmapsets/1324800#mania/2749712), which somewhat resembles a lot of big and notable features that Vide Nostalgia has, but turned up a couple notches. Easily one of his more complex and interesting maps, the map features very odd but complexing time signatures that’ll definitely catch many players off guard, whilst capturing the song through pretty complex rhythms and structures. He does all of this whilst also maintaining PR and heavy repetition when the song calls for it, making this one hell of a challenge to FC and acc for many players. Definitely give this a try if you’re looking for a challenge. 
```

Comment:
rather new ~~than~~ **compared to** some of his older mapsets

odd ~~but~~ **and** complexing

capturing the song **gracefully** through

acc >> "obtain accuracy on"